pkg/reconciler/openshift/tektontrigger/transformers.go

```diff
@@ -67,3 +68,34 @@ func replaceDeploymentArgs(argToReplace, value string) mf.Transformer {
 		return nil
 	}
 }
+
+// add key values in the default configmap for triggers for runAsUser and runAsGroup
+func addConfigMapData(cmName, key, value string) mf.Transformer {
```

Comment:
this is already generalized function, please use that

Reply:
You mean this one https://github.com/tektoncd/operator/blob/main/pkg/reconciler/common/transformers.go#L532 ?

actually i tried that but the function ignore when value is `""`
https://github.com/tektoncd/operator/blob/main/pkg/reconciler/common/transformers.go#L568-L570

and don't want to break existing functionality by doing code changes to that function so written new one considering we need this PR soon so

Reply:
@savitaashture can you try this?
```
import 	"knative.dev/pkg/ptr"

type DefaultRunAsValues struct {
	RunAsUser  *string `json:"default-run-as-user"`
	RunAsGroup *string `json:"default-run-as-group"`
}

emptyDefaultRunAsValues := DefaultRunAsValues{
	RunAsUser:  ptr.String(""),
	RunAsGroup: ptr.String(""),
}
common.AddConfigMapValues(tektontrigger.ConfigDefaults, emptyDefaultRunAsValues)
```

Reply:
I have updated with existing function with pointer